webapp/packages/core-connections/src/Administration/ConnectionsResource.ts

```diff
@@ -62,6 +62,15 @@ export class ConnectionsResource extends CachedMapResource<string, AdminConnecti
     return (connection as ConnectionNew)[NEW_CONNECTION_SYMBOL];
   }
 
+  getNewer(aId: string, bId: string): number {
+    const [a, b] = [this.get(aId), this.get(bId)] as ConnectionNew[];
+    if (!a || !b) {
+      return 0;
+    }
+
+    return b.timestamp - a.timestamp;
+  }
```

Comment:
I recommend to rewrite this and `isNew` function in functional style (make pure, accepts two connections as arguments)